runners/console/index.ts

```diff
@@ -254,6 +254,25 @@ export class Console extends Runner {
         return result;
     }
 
+    runDownloadFile(step: Step, command: Command): RunResult {
+        let result = new RunResult();
+        result.returnCode = 0;
+
+        let installDir = this.getVariable(this.workspaceDirectory);
+        if (command.parameters.length == 3) {
+            installDir = path.join(installDir, command.parameters[2]);
+            this.createFolder(installDir, false);
+        }
+        var command1;
+        if(this.platform == ConsolePlatform.WINDOWS) {
+            command1 = "powershell.exe \"Invoke-WebRequest -OutFile " +   command.parameters[1] + " '" + command.parameters[0] + "'\"";
+        } else {
+            command1 = "wget -c " + command.parameters[0] + " -O " + command.parameters[1];
+        }
```

Comment:
You can write this shorter using the ternary operator.
let command = (this.platform == ConsolePlatform.WINDOWS) ? ... : ...